monkey/infection_monkey/transport/http.py

```diff
@@ -115,9 +115,6 @@ def log_message(self, format_string, *args):
 
 class HTTPConnectProxyHandler(http.server.BaseHTTPRequestHandler):
     timeout = 30  # timeout with clients, set to None not to make persistent connection
-    proxy_via = (
```

Comment:
This may be necessary if it's inherited from `http.server.BaseHTTPRequestHandler`.

Reply:
Tests pass, so, I guess this is fine.